src/main/java/com/techsophy/tsf/account/changelog/AddTheme.java

```diff
@@ -0,0 +1,39 @@
+package com.techsophy.tsf.account.changelog;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.shaded.json.parser.ParseException;
+import com.techsophy.tsf.account.entity.ThemesDefinition;
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import java.io.IOException;
+import java.io.InputStream;
+import static com.techsophy.tsf.account.constants.AccountConstants.*;
+import static com.techsophy.tsf.account.constants.ErrorConstants.EXCEUTION_IS_FAILED;
+
+@ChangeUnit(id = ADD_THEME, order = ORDER_4, systemVersion = SYSTEM_VERSION_1)
+@AllArgsConstructor(onConstructor_ = {@Autowired})
+@Slf4j
+public class AddTheme {
+    private  final MongoTemplate template;
+    private final ObjectMapper objectMapper;
+
+    @Execution
+    public void changeSetFormDefinition() throws IOException, ParseException {
+        String pathUser = TP_TEMPLATE_JSON;
+        InputStream inputStreamTest = new ClassPathResource(pathUser).getInputStream();
+        ThemesDefinition themesDefinition = objectMapper.readValue(inputStreamTest, ThemesDefinition.class);
```

Comment:
Adding the theme and mapping it to user should be part of same changelog.
Theme and user mapping should only be done when there are no records in the system.